src/test/java/org/prebid/server/floors/PriceFloorsConfigResolverTest.java

```diff
@@ -0,0 +1,256 @@
+    }
+
+    @Test
+    public void updateFloorsConfigShouldReturnDefaultConfigIfEnforceFloorsRateMoreThanMaximumValue() {
+        // given
+        final Account givenAccount = Account.builder()
+                .id("some-id")
+                .auction(AccountAuctionConfig.builder()
+                        .priceFloors(AccountPriceFloorsConfig.builder()
+                                .enforceFloorsRate(101).build())
+                        .build())
+                .build();
+
+        // when
+        final Future<?> future = testingInstance.updateFloorsConfig(givenAccount);
+
+        // then
+        assertThat(future.succeeded()).isTrue();
+        assertThat(future.result())
+                .isEqualTo(withDefaultFloorsConfig().toBuilder().id("some-id").build());
+        verify(metrics).updateAlertsConfigFailed("some-id", MetricName.price_floors);
+    }
+
+    @Test
+    public void updateFloorsConfigShouldReturnDefaultConfigIfPeriodicSecLessThanMinimumValue() {
+        // when
+        final Future<?> future = testingInstance.updateFloorsConfig(
+                accountWithFloorsFetchConfig(config -> config.periodSec(200L)));
+
+        // then
+        assertThat(future.succeeded()).isTrue();
+        assertThat(future.result())
+                .isEqualTo(withDefaultFloorsConfig().toBuilder().id("some-id").build());
+        verify(metrics).updateAlertsConfigFailed("some-id", MetricName.price_floors);
+    }
+
+    @Test
+    public void updateFloorsConfigShouldReturnDefaultConfigIfPeriodicSecMoreThanMaxAgeSec() {
+        // when
+        final Future<?> future = testingInstance.updateFloorsConfig(accountWithFloorsFetchConfig(config ->
+                config.periodSec(900L).maxAgeSec(800L)));
+
+        // then
+        assertThat(future.succeeded()).isTrue();
+        assertThat(future.result())
+                .isEqualTo(withDefaultFloorsConfig().toBuilder().id("some-id").build());
+        verify(metrics).updateAlertsConfigFailed("some-id", MetricName.price_floors);
+    }
+
+    @Test
+    public void updateFloorsConfigShouldReturnDefaultConfigIfMaxAgeSecLessThanMinimumValue() {
+        // when
+        final Future<?> future = testingInstance.updateFloorsConfig(
+                accountWithFloorsFetchConfig(config -> config.maxAgeSec(500L)));
+
+        // then
+        assertThat(future.succeeded()).isTrue();
+        assertThat(future.result())
+                .isEqualTo(withDefaultFloorsConfig().toBuilder().id("some-id").build());
+        verify(metrics).updateAlertsConfigFailed("some-id", MetricName.price_floors);
+    }
+
+    @Test
+    public void updateFloorsConfigShouldReturnDefaultConfigIfMaxAgeSecMoreThanMaximumValue() {
+        // when
+        final Future<?> future = testingInstance.updateFloorsConfig(
+                accountWithFloorsFetchConfig(config -> config.maxAgeSec(Integer.MAX_VALUE + 1L)));
+
+        // then
+        assertThat(future.succeeded()).isTrue();
```

Comment:
Here and in other cases, as far as I have understood `updateFloorsConfig` returns `succeededFuture` anyways, so why do we need this check here? Is there any option that we will get failedFuture? If not, this check is redundant.